lms/djangoapps/edxnotes/helpers.py

```diff
@@ -59,12 +59,7 @@ def get_edxnotes_id_token(user):
         raise ImproperlyConfigured(
             'OAuth2 Client with name [{}] does not exist.'.format(CLIENT_NAME)
         )
-
-    scopes = ['email', 'profile']
-    expires_in = settings.OAUTH_ID_TOKEN_EXPIRATION
-    jwt = JwtBuilder(user, secret=client.client_secret).build_token(scopes, expires_in, aud=client.client_id)
-
-    return jwt
+    return create_jwt_for_user(user, secret=client.client_secret, aud=client.client_id)
```

Comment:
The JWT returned here will no longer include the user's email. I suspect you plan to resolve that in a follow up PR.

Reply:
`create_jwt_for_user` does include user's email and profile.  See this [docstring](https://github.com/edx/edx-platform/blob/f5a643f53b2ec41f564425c47e23dbf793febac1/openedx/core/djangoapps/oauth_dispatch/jwt.py#L89) and [code](https://github.com/edx/edx-platform/blob/f5a643f53b2ec41f564425c47e23dbf793febac1/openedx/core/djangoapps/oauth_dispatch/jwt.py#L103).  This module simply creates and returns a `jwt` based on what's passed in.

The issue that you're referring to is the code that goes through the underlying OAuth layer in [create_dot_access_token](https://github.com/edx/edx-platform/blob/f5a643f53b2ec41f564425c47e23dbf793febac1/openedx/core/djangoapps/oauth_dispatch/api.py#L25) and [refresh_dot_access_token](https://github.com/edx/edx-platform/blob/f5a643f53b2ec41f564425c47e23dbf793febac1/openedx/core/djangoapps/oauth_dispatch/api.py#L48).  Yes, that is a different issue that I'm now looking into.  Although I don't believe it's a regression since the mobile apps don't require the email/profile scopes.